packages/@o3r/stylelint-plugin/schematics/ng-add/index.ts

```diff
@@ -32,7 +33,7 @@ function ngAddFn(options: NgAddSchematicsSchema): Rule {
       setupDependencies
     } = await import('@o3r/schematics');
     // eslint-disable-next-line @typescript-eslint/naming-convention
-    const { NodeDependencyType } = await import('@schematics/angular/utility/dependencies');
+    const { NodeDependencyType } = await import('@schematics/angular/utility/dependencies').catch(() => ({ NodeDependencyType: { Dev: 'devDependencies' as NodeDependencyType.Dev } }));
```

Comment:
We should probably avoid shadowing the type `NodeDependencyType` with the variable of the same name

Reply:
Actually only the string value is used on this one but changing the interface would be a breaking change :S

Reply:
I'm just talking about renaming the const, how would that be a breaking change?

Reply:
Oooh sorry, I just understand your point.
Actually there are the same from the same import except that on is a type in sync and the impl in dynImport.
renaming one or the other would be almost more confusing, don't you think?
(The linter does not complain)

Reply:
The new one maybe ! :)